tests/gis_tests/rasterapp/models.py

```diff
@@ -1,8 +1,34 @@
 from django.contrib.gis.db import models
 from django.contrib.gis.gdal import HAS_GDAL
+from django.db import connections
+
+
+class SkipChecksIfNotDBFeaturesMixin(object):
+    """
+    Required because we make the models but then skip the tests for them
```

Comment:
I'd say, "Skip system checks on models that don't have all the required_db_features."